templates/part.card.php

```diff
@@ -28,54 +28,45 @@
-			<ui-select-match placeholder="Select labels…">
+			<ui-select-match placeholder="<?php p($l->t('Select labels…')); ?>">
 				<span class="select-label" style="background-color:#{{$item.color}}; color:{{ $item.color|textColorFilter }};">{{$item.title}}&nbsp;</span>
 			</ui-select-match>
 			<ui-select-choices
 				repeat="label in boardservice.getCurrent().labels | filter:$select.search">
-				<span
-					style="background-color:#{{label.color}}; color:{{ label.color|textColorFilter }};">{{label.title}}</span>
+				<span class="choose-label" style="background-color:#{{label.color}}; color:{{ label.color|textColorFilter }};">{{label.title}}</span>
 			</ui-select-choices>
 		</ui-select>
-
-		<div class="duedate">
-			<input class="datepicker-input medium focus" type="text" placeholder="Set a due date" value="{{ cardservice.getCurrent().duedate | parseDate }}" datepicker="due" />
-			<input class="timepicker-input medium focus" type="text" placeholder="00:00:00" ng-if="cardservice.getCurrent().duedate" value="{{ cardservice.getCurrent().duedate | parseTime }}" timepicker="due" />
-			<button class="icon icon-delete button-inline" title="<?php p($l->t('Remove due date')); ?>" ng-if="cardservice.getCurrent().duedate" ng-click="resetDuedate()"></button>
-		</div>
-
+	</div>
+	<h3>
+		<?php p($l->t('Due date')); ?>
+	</h3>
+	<div class="duedate">
+		<input class="datepicker-input medium focus" type="text" placeholder="<?php p($l->t('Click to set')); ?>" value="{{ cardservice.getCurrent().duedate | parseDate }}" datepicker="due" />
+		<input class="timepicker-input medium focus" type="text" placeholder="00:00:00" ng-if="cardservice.getCurrent().duedate" value="{{ cardservice.getCurrent().duedate | parseTime }}" timepicker="due" />
+		<button class="icon icon-delete button-inline" title="<?php p($l->t('Remove due date')); ?>" ng-if="cardservice.getCurrent().duedate" ng-click="resetDuedate()"></button>
 	</div>
 
-	<!--<div id="assigned-users">
-		<ui-select multiple tagging="" ng-model="card.assignees"
-				   theme="bootstrap" style="width:100%;"
-				   title="Choose a user to assign"
-				   placeholder="Assign users…"
-				   on-select="userAssign($item, $model)"
-				   on-remove="userRemove($item, $model)" ng-disabled="archived">
-			<ui-select-match placeholder="Select users…">{{$item.title}}
-			</ui-select-match>
-			<ui-select-choices
-				repeat="label in boardservice.getCurrent().labels | filter:$select.search">
-				<span
-					style="background-color:#{{label.color}}">{{label.title}}</span>
-			</ui-select-choices>
-		</ui-select>
-	</div>//-->
-
+	
 	<div id="card-description">
 		<h3>
-			<?php p($l->t('Description')); ?>
-			<a href="https://github.com/nextcloud/deck/wiki/Markdown-Help" target="_blank" class="icon-help" title="<?php p($l->t('Formatting help')); ?>"></a>
-			<span class="save-indicator"><?php p($l->t('Saved')); ?></span>
+			<div>
```

Comment:
But we can just apply the flex styles to `#card-meta h3` or am I missing something?